.github/workflows/_test_container.yml

```
@@ -0,0 +1,139 @@
+          # Ensure the binary is executable
+          chmod +x ${{ inputs.binary_output_path }}
+          # Verify binary works
+          ${{ inputs.binary_output_path }} version
+
+      - name: Run tests
+        shell: bash
+        working-directory: ${{ inputs.test_working_dir }}
+        run: |
+          # Verify binary works before running tests
+          common_assets/bacalhau_bin version
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)